cmd/podman/root.go

```diff
@@ -404,25 +430,20 @@ func stdOutHook() {
 }
 
 func rootFlags(cmd *cobra.Command, podmanConfig *entities.PodmanConfig) {
-	srv, uri, ident, machine := resolveDestination()
-
 	lFlags := cmd.Flags()
 
-	// non configurable option to help ssh dialing
-	podmanConfig.MachineMode = machine
-
 	sshFlagName := "ssh"
 	lFlags.StringVar(&podmanConfig.SSHMode, sshFlagName, string(ssh.GolangMode), "define the ssh mode")
 	_ = cmd.RegisterFlagCompletionFunc(sshFlagName, common.AutocompleteSSH)
 
 	connectionFlagName := "connection"
-	lFlags.StringP(connectionFlagName, "c", srv, "Connection to use for remote Podman service")
+	lFlags.StringP(connectionFlagName, "c", "", "Connection to use for remote Podman service")
```

Comment:
Why are we hiding this information from the user.  It is useful, when the user executes podman --help

Reply:
Thanks Dan! I've added it back with some refactored code.
FYI @Luap99